### ISSUE TITLE
27.1.0 build broken on Mac build

### ISSUE THREAD

**Issue by stevenschlansker** [MoonchildProductions/Pale-Moon#927]
The 27.1.0 Mac build is currently failing:

```
27:45.05 In file included from tycho/default/ws/obj-x86_64-apple-darwin12.5.0/dom/media/platforms/Unified_cpp_dom_media_platforms0.cpp:29:

27:45.06 tycho/default/ws/dom/media/platforms/apple/AppleATDecoder.cpp:496:21:
error: no matching function for call to 'AudioFileStreamParseBytes'

27:45.07   OSStatus status = AudioFileStreamParseBytes(mStream,
```

Hopefully not a terribly hard issue to track down; I'll work on it over the next few days.  (can't assign it to myself)

**Comment by wolfbeast** [comment]
Assigned you

**Comment by trav90** [comment]
@stevenschlansker try building with this extra commit: ab938f4

**Comment by stevenschlansker** [comment]
Thanks @trav90 that seems to help.  There's more build errors down the line though, so I will track those down and make sure the build succeeds before closing this one for sure.

**Comment by trav90** [comment]
This should be fixed by #955
